test/nodes/test_connector.py

```diff
@@ -161,3 +168,17 @@ def test_crawler_extract_hidden_text(test_url, tmp_path):
     )
     crawled_content = documents["documents"][0].content
     assert "hidden text" not in crawled_content
+
+
+def test_crawler_loading_wait_time(test_url, tmp_path):
```

Comment:
By the way, thank you for the test! Well done. I wonder if we could extend it by making the existing `page_dynamic.html` lose a link and also add another one. Just to make sure it all works in both directions :slightly_smiling_face: